src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -80,6 +80,36 @@ public static function castTimeZone(\DateTimeZone $timeZone, array $a, Stub $stu
         return $filter & Caster::EXCLUDE_VERBOSE ? $z : $z + $a;
     }
 
+    public static function castPeriod(\DatePeriod $p, array $a, Stub $stub, $isNested, $filter)
+    {
+        if (defined('HHVM_VERSION_ID') || \PHP_VERSION_ID < 50605) {
+            return $a;
+        }
+
+        $dates = array();
+        if (\PHP_VERSION_ID >= 70107) { // see https://bugs.php.net/bug.php?id=74639
+            foreach (clone $p as $i => $d) {
+                $dates[] = sprintf('%s) %s', $i + 1, $d->format('Y-m-d H:i:s'));
+                if (11 === $i) {
+                    $dates[] = '...';
```

Comment:
I added ``x more`` ;) I compared ``iterator_count()`` and my method with 1M of reccurences:
- 4160.1 ms with ``iterator_count()``
- 0.1 ms with my method